src/pages/messaging/customer-chat-api/v3.4/index.mdx

```diff
@@ -157,11 +157,11 @@ It returns [summaries](/messaging/customer-chat-api/v3.4/data-structures/#chat-s
 <CodeSample path={'REQUEST'}>
 
 ```shell
-curl -X POST \
-  https://api.livechatinc.com/v3.4/customer/action/list_chats?organization_id=<organization_id> \
-  -H 'Content-Type: application/json' \
-  -H 'Authorization: Bearer <customer_access_token>' \
-  -d '{}'
+curl --location --request POST \
+'https://api.livechatinc.com/v3.4/customer/action/list_chats?organization_id=<organization id>' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <customer access token>' \
+--data-raw '{}'
 ```
```

Comment:
I saw that there's an error when importing this curl into Postman - the only thing I changed is I put the URL into
`'https://api.livechatinc.com/v3.4/customer/action/list_chats?organization_id=<organization id>'`
and it imported alright - so I don't think we should change -H into --header, add the --location and --request, and so on since it won't be consistent with other curls. I'd leave just the '' for the URL

Reply:
Thank you for the feedback, I have adjusted accordingly